test/benchee/statistics_test.exs

```diff
@@ -57,6 +67,8 @@ defmodule Benchee.StatistcsTest do
       stats
     end
 
+    # no mode asserts in the sample asserts as the samples have no duplicated
+    # values and therefore mode isn't a useful value
```

Comment:
This is an interesting edge case, actually. In this case, do we want to avoid returning a mode at all if there are no duplicated values? And what about in cases like `[45, 45, 55, 55, 40, 50, 60]`? What do we pick for a mode there, or do we not have a mode at all?

EDIT:

After consulting the good folks at Wikipedia, it seems like the correct thing to do might be to have _two_ modes in that example I gave. They say:
> Given the list of data [1, 1, 2, 4, 4] the mode is not unique – the dataset may be said to be bimodal, while a set with more than two modes may be described as multimodal.